docs/install.md

```diff
@@ -144,7 +156,7 @@ pip install -v -e .
 pip install mmpose
```

Comment:
`pip install git+https://github.com/open-mmlab/mmpose.git`

Reply:
Addressed in https://github.com/open-mmlab/mmhuman3d/pull/237/commits/317ecc4ec07cd447693255294d09cabe8ccbe2bc.